packages/py-api/py_api/middleware/auth.py

```diff
@@ -22,17 +22,15 @@ class AuthMiddleware:
         "/health": ["GET"], "/routes": ["GET"], "/fswitches": ["GET"],
     }
 
-    @classmethod
-    def bind(cls, app: FastAPI) -> None:
-
+    def __init__(self, app: FastAPI) -> None:
```

Comment:
Why did you decide to have a constructor? Do we need an instance?

Reply:
So that it matches the approach we use for routes. I decided that only the binding class should have a method bind.

Reply:
Nice one